Assets/Scripts/Player/PlayerManager.cs

```diff
@@ -1,43 +1,94 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
-using Inputs;
 using Common;
+using BTManager;
+using System;
 
 namespace Player
 {
+    [System.Serializable]
+    public class PositionData
```

Comment:
if u just need to store data use a struct instead of class

Reply:
Fixed